assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -159,12 +160,29 @@ export const selectors = {
 	 * @since 1.7.0
 	 *
 	 * @param {Object} state Data store's state.
+	 * @param {string} page Page query argument to add to admin URL.
```

Comment:
This should use `{(string|undefined)}` for the type, and per our (and WordPress) docs conventions there should be an "Optional. " at the beginning of the description. It would also be good to add something like "If not provided, the base admin URL is returned.".

Reply:
Thanks! Updated.